extranet/src/SignupForm.tsx

```diff
@@ -47,6 +71,17 @@ const SignupForm: React.FC = () => {
   } = useForm<FormData>();
 
   const classes = useStyles();
```

Comment:
```suggestion
  const { container, imageContainer, image, formContainer, bar, button } = useStyles();
```

This will allow you to use the style easily and in a proper way

Reply:
it's done